app/models/akka/RequesterBehavior.scala

```diff
@@ -30,8 +31,8 @@ trait RequesterBehavior { me: AbstractDSLinkActor with Meter =>
   ), "stateKeeper")
 
   // used by Close and Unsubscribe requests to retrieve the targets of previously used RID/SID
-  private val targetsByRid = collection.mutable.Map.empty[Int, String]
-  private val targetsBySid = collection.mutable.Map.empty[Int, PathAndQos]
+  private var targetsByRid = collection.mutable.Map.empty[Int, String]
+  private var targetsBySid = collection.mutable.Map.empty[Int, PathAndQos]
 
```

Comment:
var and mutable map. Should we use val?

Reply:
Nope. We replace this state in case of recovery at 139-140 lines.